src/display/editor/toolbar.js

```diff
@@ -123,7 +123,7 @@ class EditorToolbar {
     const { editorType, _uiManager } = this.#editor;
 
     const button = document.createElement("button");
-    button.className = "delete";
```

Comment:
It looks like this rename is making integration tests fail because they rely on the existing name.